e2e/cypress/integration/system_console/ui_and_api/customization_spec.js

```diff
@@ -54,4 +58,31 @@ describe('Customization', () => {
             expect(config.TeamSettings.SiteName).to.eq(siteName);
         });
     });
+
+    it('SC20337 Can change Support Email setting', () => {
```

Comment:
Probably just my ignorance, but why can't this be a regular unit test rather than an cypress test?

Reply:
This is the first phase of system console tests - to verify that the UI is able to save config change into the server/database. Some of the actions here are scrolling, entering input, clicking on enabled save button (where if no change, it's disabled and cannot be clicked) that may/may not work depending on element visibility, state, config representation, etc.
Next phase is to verify the effect of each config in the UI itself, most likely in the team/channel view.
I guess the advantage here is it simulates user interaction and dealing with real data/info.

Reply:
I get why we are writing tests, but shouldn't we prefer unit tests to cypress tests?

Reply:
Maybe this test is doing more than it should be.  Probably, checks to label or help-text should be moved into unit test but IMO, the rest is appropriate for E2E. I could be totally wrong here but that's what I thought we should have in terms of E2E.

Reply:
Ok. As long as we are making a conscious decision to do this.

Reply:
Yup, this campaign has prior discussion/agreement in QA meeting before releasing as HW and have [entries in the release testing spreadsheet](https://docs.google.com/spreadsheets/d/1thMNF1gD6DRwbdAaKF4mrJpmDLRzFAk45QSEPvhLUbw/edit#gid=190688202&range=305:318)